docs/evm/evm-simple.md

```diff
@@ -0,0 +1,92 @@
+---
+id: evm-simple
+title: A simple EVM script and NEAR CLI interaction
+sidebar_label: Simple script & NEAR     CLI guide
```

Comment:
Is the extended space in "NEAR     CLI guide" intentional?

Reply:
no good eye! thank you